tutorial/sample-problems/Geometry/LineSegmentGraphTool.pg

```diff
@@ -0,0 +1,65 @@
+## DESCRIPTION
+## Shows a triangle tool within the GraphTool.
+## ENDDESCRIPTION
+
+## DBsubject(WeBWorK)
+## DBchapter(Sample Problems)
+## DBsection(Graph Tool)
+## Date(02/25/2025)
+## Institution(Missouri Western)
+## Author(Glenn Rice)
+## MO(1)
+## KEYWORDS('graph tool', 'vectors')
+
+#:% name = Line Segment Graph Tool
+#:% type = Sample
+#:% subject = [geometry]
+#:% categories = [graph, graphtool]
+#:% see_also = [VectorGraphTool.pg, TriangleGraphTool.pg, QuadrilateralGraphTool.pg]
+
+#:% section = preamble
+#: Load the PODLINK('parserGraphTool.pl') macro to be able to use the GraphTool.
+DOCUMENT();
+
+loadMacros('PGstandard.pl', 'PGML.pl', 'parserGraphTool.pl', 'PGcourse.pl');
+
+#:% section = setup
+#: This problem first finds two points with unequal x-values and unequal y-values.
```

Comment:
This should say "This problem first finds two distinct points."  The problem in fact does not find two points with unequal x and unequal y values.  The x-values may be equal, and in that case the y-values must be different, and vice versa.